index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Winston Churchill Page</title>
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+
+</head>
+
+<body>
+
+
+<h1> History will be <span.bold>kind</span.bold> to me </h1>
```

Comment:
:hammer: `<span.bold>` isn't the right format. We need a class attribute for HTML and we use the dot selectors for classes in CSS.